src/sass/_button.scss

```diff
@@ -0,0 +1,13 @@
+.sk-button {
+  display: block;
+  text-align: center;
+  font-weight: 600;
+  width: 100%;
+  padding: 1.5rem 1rem;
+  margin-bottom: 1rem;
+  border: none;
+  color: white;
+  background-color: blue;
+  cursor: pointer;
+  background-color: #1aa5a8;
+}
```

Comment:
Nit: Should probably have a trailing newline

Reply:
I introduced prettier